modules/ROOT/pages/deploying-to-multiple-environments.adoc

```diff
@@ -0,0 +1,234 @@
++
+image::deploying-to-multiple-environments-06.png[]
+
+. You can change the environment by editing the configured *Global Property* value.
+
+=== Using XML Editor
+
+. Define a list of environments you wish to support in your application. For example, one common use case involves configuring the application to support both Production and a QA environments. +
+We will define the variable `variable` in the environments `prod` and `qa` and test it using a single application.
+
+[start=2]
+. Right-click the  `src/main/resources`  folder, then select  *New*  >  *File*   to create a properties file for each environment you wish to support. For example:
+** `qa.yaml`
+** `prod.yaml`
++
+image::deploying-to-multiple-environments-01.png[]
+
+. Double-click each of these new files to open them in new tabs in Studio. By editing them, you can configure the properties of the environment that correspond to the filename. For example, you may wish to add the properties as per the following images. Keep in mind that you can use these properties anywhere in your application.
++
+image::deploying-to-multiple-environments-02.png[]
++
+image::deploying-to-multiple-environments-03.png[]
+
+. Close the properties files.
+
+. Click in the Configuration XML tab of your Studio application, below the canvas.
+
+. Add a *configuration-properties* element to the XML file and let the file name configurable.
+
+[source,xml,linenums]
+----
+<configuration-properties doc:name="Configuration properties" file="${env}.yaml" />
+----
+
+[TIP]
+====
+Learn more about xref:configuring-properties.adoc[configuring variable properties].
+====
+
+[start=7]
+. Add a *Global Property* element to set the default environment name.
+
+[source,xml,linenums]
+----
+<global-property doc:name="Global Property" name="env" value="qa" />
+----
+[start=8]
+. You can change the environment by editing the configured *Global Property* value.
+
+
+== Testing Configured Environment Properties in Studio
+
+You can test that your application uses the specific environment you configured in your variables properties, by running the application in Mule embedded in Studio.
+
+. In the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Mule Application*.
+
+. Studio automatically deploys your application according to the environment variable you specified in the *Global Property* from the previous procedure.
+
+. Again in the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Run Configurations...*.
++
+image::deploying-to-multiple-environments-07.png[]
+
+. Define the System Property in the `Arguments` tab.
++
+image::deploying-to-multiple-environments-08.png[]
+
+. Click the `Run` button to complete the procedure. The properties configured in `prod.yaml` will be used instead of the ones defined in `qa.yaml`.
+
+== Deploying Mule Apps using your Specific Environment Configurations
```

Comment:
Title-style capitalization capitalizes Using and Your as well.

See https://confluence.internal.salesforce.com/display/public/DOCSTYLE/title+capitalization